src/components/follow-suggestion/follow-suggestion.css

```diff
@@ -0,0 +1,105 @@
+.follow__conatiner {
+    position: sticky;
+    top: 1rem;
+    width: 100%;
+    background-image: linear-gradient(to right bottom, var(--secondry-light-color), var(--white-color));
+    min-height: 37rem;
+    padding: 1rem;
+    border-radius: 1.2rem;
+}
+
+.search__div {
+    width: 100%;
+    position: relative;
+}
+
+.search__div > input {
+    position: sticky;
+    top: 1rem;
+    width: 100%;
+    padding: .7rem 4rem;
+    font-size: 1.3rem;
+    outline: none;
+    border-bottom: 2px solid var(--primary-color);
+    border-radius: 1.2rem;
+    background-color: var(--white-color);
+}
+
+.search__div > i{
+    position: absolute;
+    z-index: 11;
+    font-size: 2rem;
+    top: .8rem;
+    left: 1rem;
+    color: var(--primary-color);
+}
+
+.follow__conatiner > header {
+    border-bottom: 1px solid var(--white-color);
+    padding: 1rem 0rem;
+}
+
+.follow__conatiner > header > p {
+    font-size: 1.6rem;
+    font-weight: bolder;
+    font-family: sans-serif;
+}
+
+.profile__container {
+    display: flex;
+    flex-wrap: wrap;
+    border-bottom: 1px solid var(--white-color);
+    padding: 1.5rem 0rem;
+    cursor: pointer;
+}
+
+.profile__img {
+    height: 5rem;
+    width: 5rem;
+    border-radius: 50%;
+    background-color: var(--white-color);
+}
+
+.profile__img > img {
+    max-width: 100%;
+    max-height: 100%;
+}
+
+.profile__name {
+    margin-left: 1rem;
+}
+
+.name {
+    font-size: 1.5rem;
+    font-weight: 800;
+}
+
+.id {
+    font-size: 1.2rem;
+    color: var(--secondry-text-color);
+}
+
+.follow__btn {
+    font-size: 1.5rem;
+    padding: .5rem 1rem;
+    background-color:var(--primary-color);
+    border-radius: 1.2rem;
+    align-self: center;
+    margin-left: auto;
+    color: white;
+    font-weight: 800;
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+}
+
+.see__more {
+    font-size: 1.8rem;
+    margin: 1.5rem 0rem;
+    text-decoration: underline;
+    text-underline-offset: 2px;
+    color: var(--primary-color);
+    cursor: pointer;
```

Comment:
small thing but cursor pointer is in entire container, you can either center the see more or hit pointer only on text